coriolisclient/v1/logging.py

```diff
@@ -125,7 +126,7 @@ def _convert_period_to_timestamp(self, period):
         try:
             _ = datetime.datetime.fromtimestamp(int(period))
             return int(period)
-        except:
+        except Exception:
             pass
```

Comment:
Let's please make this code more testable. 
First, let's find out what exceptions `fromtimestamp` could raise: https://docs.python.org/3/library/datetime.html#datetime.date.fromtimestamp
And also, `int()` can raise `ValueError`. We'll need these to log warnings properly (instead of just passing).
It should look something like this:
```suggestion
        except OverflowError, OSError:
            LOG.warning("Failed to initialize timestamp from period value: %d", period)
        except ValueError:
            LOG.warning("Invalid value type for period: %s", period)
```